plugins/coreboot/cbfs_serialized.h

```diff
@@ -0,0 +1,240 @@
+/*
+ * This file is part of the coreboot project.
```

Comment:
I thought we were going to use the "installed" headers in the `coreboot-utils` package?

Reply:
Yes, but until that is available on my test machine I'm using the headers.